src/hypothesis/internal/conjecture/utils.py

```diff
@@ -290,6 +290,7 @@ def __init__(self, data, min_size, max_size, average_size):
         self.rejections = 0
         self.drawn = False
         self.force_stop = False
+        self.freebies = 0
```

Comment:
While this looks reasonable to me, I think [it might have broken something](https://travis-ci.org/HypothesisWorks/hypothesis-python/jobs/269131370).